easybuild/easyconfigs/d/dune-fem/dune-fem-2.8.0.6-foss-2020b.eb

```diff
@@ -0,0 +1,67 @@
+easyblock = 'PythonBundle'
+
+name = 'dune-fem'
+version = '2.8.0.6'
+
+homepage = "https://www.dune-project.org/modules/dune-fem/"
+description = """DUNE-FEM is a discretization module based on DUNE containing all
+the building blocks required to implement efficient solvers for a wide range
+of (systems of non linear) partial differential equations. DUNE-FEM can also
+be used through an extensive Python interface which brings all
+components of DUNE-FEM and the DUNE core modules to Python.
+"""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+
+dependencies = [
+    ('Python', '3.8.6'),
+    ('SciPy-bundle', '2020.11'),
+    ('PETSc', '3.14.4'),
+    ('CMake', '3.18.4'),
```

Comment:
also here, move CMake to the builddependencies?

Reply:
It is required at buildtime and runtime, see above.